.github/workflows/ci.yml

```diff
@@ -70,13 +66,11 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Install
-        run: npm install
+      - name: Install with legacy peer deps
+        run: npm install --legacy-peer-deps
+      
+      - name: Compile TS
+        run: tsc lib/**/*.ts
```

Comment:
How is this library published? We'll just want to make sure we don't accidentally not perform the compilation step. I think there's a prepare or prepublish script we can invoke in package.json to minimize errors.